frontend/apps/web/src/pages/index.tsx

```diff
@@ -1,151 +1,188 @@
-import { RepositoryData } from '@/mocks'
+import { PageContainer } from '@/containers/PageContainer'
 import {
   Button,
-  Icon,
-  NativeLink,
+  Colors,
+  Link,
   PageLayout,
-  StackLayout,
+  ProgressBar,
   Table,
   Text,
-  ProgressBar,
   usePageLayout,
 } from '@filecoin/ui'
-import React, { useState } from 'react'
-import { NavLink, useNavigate } from 'react-router-dom'
+import { styled } from '@storybook/theming'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
 
 const Header = props => {
-  const [activeTab, setActiveTab] = useState(1)
-
+  const navigate = useNavigate()
   return (
     <PageLayout.Header>
-      <StackLayout>
+      <HeaderWrapper>
         <Text type="heading 5">Systems</Text>
-      </StackLayout>
-      <PageLayout.Tabs>
-        <NavLink to={'/system/overview'}>
-          <PageLayout.Tab
-            onClick={() => setActiveTab(1)}
-            active={activeTab === 1}
+        <Buttons>
+          <Button
+            onClick={() => navigate('/repository-details')}
```

Comment:
Add routes
for tests `/tests`
for behaviors `/behaviors`

Reply:
Done